plugins/chat/assets/javascripts/discourse/services/chat-tracking-state-manager.js

```diff
@@ -76,7 +76,7 @@ export default class ChatTrackingStateManager extends Service {
 
   get allChannelUrgentCount() {
     return (
-      this.publicChannelMentionCount +
+      this.allChannelMentionCount +
```

Comment:
This allows us to show the urgent badge on the chat header icon when a user is mentioned within a thread in a DM channel.